src/adapters/CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
```

Comment:
This changelog will only be relevant for adapters, so maybe language like "Changes to adapters wil lbe documented in this file"